src/Application.php

```diff
@@ -20,6 +20,13 @@ public function __construct(array $config, Logger $logger, ?ConfigurationInterfa
     {
         parent::__construct();
 
+        if (!empty($config['image_parameters']['approveHostnames'])) {
+            $this->validateHostname(
+                $config['image_parameters']['approveHostnames'],
```

Comment:
`approveHostnames` -> `approvedHostnames` ... tak je to gramaticky spravne a aj v priklade od Vlada v Jire.

Reply:
Prazdne pole by nemalo nikdy prejst ... tj. ak kluc neexistuje, tak nekontrolujem ... ak prazdne pole, tak validujem normalne -> neprejde to .... kedze v Dev Portali bude nastavene prazdne pole `[]` ... a v sync scripte sa nastavia konkretne hosty (ako sme rozpravali) ... teda ak by to niekto napr. omylom zo sync scriptu zmazal, tak to spadne, ...  a nie povoli vsetky hosty ... oprav to prosim a pridaj test.

Reply:
prázdné pole nikdy neprojde, protože teď to je udělané tak že pokud ten klíč existuje tak se kontroluje jestli v tom poli existuje host/port - doplním k tomu test

Reply:
ok, dopln test. tu riesis `!empty($config['image_parameters']['approveHostnames']` teda ak `approvedHostnames = []`, tak IMHO sa kontrola nespusti ... lebo `empty([]) = true`

Reply:
tak teď by to mělo být v pohodě :)